packages/open-next/src/converters/aws-apigw-v2.ts

```diff
@@ -97,11 +97,11 @@ function convertToApiGatewayProxyResultV2(
         ),
     )
     .forEach(([key, value]) => {
-      if (value === null) {
+      if (value === null || value === undefined) {
         headers[key] = "";
         return;
       }
-      headers[key] = Array.isArray(value) ? value.join(", ") : value.toString();
+      headers[key] = Array.isArray(value) ? value.join(", ") : `${value}`;
```

Comment:
I think this is the same thing. What made you change this? Is it that sometimes value does not have a toString method?

Reply:
Yea, exactly. To be safe.